kolibri/plugins/example_plugin/example_plugin_stats.json

```diff
@@ -0,0 +1 @@
+{"status":"done","chunks":{"example_plugin":[{"name":"example_plugin-4ab738054d0a0dd9389c.js","publicPath":"kolibri.plugins.example_plugin/example_plugin-4ab738054d0a0dd9389c.js","path":"kolibri/plugins/example_plugin/static/kolibri.plugins.example_plugin/example_plugin-4ab738054d0a0dd9389c.js"}]},"publicPath":"kolibri.plugins.example_plugin/"}
```

Comment:
Must have been committed by accident? I think we'll want to add these files to our gitignore.